app/manifest.json

```diff
@@ -42,15 +42,7 @@
       "run_at": "document_idle"
     }
   ],
-  "applications": {
-    "gecko": {
-      "id": "{c1687a9a-9054-430e-94cf-2ef9b3caeb7b}",
-      "update_url": "https://raw.githubusercontent.com/fasterize/fasterize-web-extension/master/app/update-manifest.json",
-      "strict_min_version": "48.0"
-    }
-  },
   "web_accessible_resources": [
     "show_deferjs_trace.js"
-  ],
-  "incognito": "split"
```

Comment:
This code is used for incognito mode. Don't remove it.

Reply:
Ok mais cela crée des erreurs sous Firefox...

Reply:
cette clé est supprimée pour firefox lors de la release.